### PR TITLE
DVX-528 Add pyatlan to conda

### PR DESCRIPTION
**Pull request by ErnestoLoma**

### CODE REVIEW

File: .conda-envs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+requirements:
+  host:
+    - jinja2 ==3.1.4
+    - lazy_loader~ 0.4
+    - pip
+    - pydantic >=2.0.0,<3.0.0
+    - python
+    - requests >=2.24
+    - tenacity ==8.2.3
+    - urllib3 >=1.26.0,<3
```
Comment (Aryamanz29) [inline .conda-envs/meta.yaml:21]: Just one doubt here: It seems the `requirements.hosts` should match `requirements.txt` ([we should specify `lazy_loader~=0.4`](https://github.com/atlanhq/atlan-python/blob/main/requirements.txt)). 

I also noticed some extra spaces ` `, such as in `requests >=2.24` (though it still works)

File: .conda-envs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - tests.unit.model.fields
+    - tests.unit.pkg
+
+about:
+  home: "https://github.com/atlanhq/atlan-python"
+  license: Apache Software
+  license_family: APACHE
+  license_file:
+  summary: "Atlan Python Client"
+  doc_url:
```
Comment (Aryamanz29) [inline .conda-envs/meta.yaml:60]: Can we add dev portal link here? ie: https://developer.atlan.com/sdks/python

File: .conda-envs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  dev_url:
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```
Comment (Aryamanz29) [inline .conda-envs/meta.yaml:65]: Can we add our GitHub IDs as maintainers here?
eg:
- https://github.com/ErnestoLoma
- https://github.com/cmgrote
- https://github.com/Aryamanz29
- https://github.com/0xquark

File: .conda-envs/meta.yaml (hunk 1 above)
Comment (Aryamanz29) [reply .conda-envs/meta.yaml:21]: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run

File: .conda-envs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    - tests.unit
+    - tests.unit.model
+    - tests.unit.model.fields
+    - tests.unit.pkg
+
+about:
+  home: "https://github.com/atlanhq/atlan-python"
+  license: Apache Software
+  license_family: APACHE
+  license_file:
```
Comment (Aryamanz29) [inline .conda-envs/meta.yaml:58]: We can add these as well:

`license_file: LICENSE`
`license_url: https://github.com/atlanhq/atlan-python/blob/main/LICENSE`

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#about-section

File: .conda-envs/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    - tenacity ==8.2.3
+    - urllib3 >=1.26.0,<3
+  run:
+    - jinja2 ==3.1.4
+    - lazy_loader~ 0.4
+    - pydantic >=2.0.0,<3.0.0
+    - python
+    - requests >=2.24
+    - tenacity ==8.2.3
+    - urllib3 >=1.26.0,<3
```
Comment (Aryamanz29) [inline .conda-envs/meta.yaml:29]: Same doubt here ☝️(similar to: https://github.com/atlanhq/atlan-python/pull/371/files#r1715331215)

File: .conda-envs/meta.yaml (hunk 4 above)
Comment (ErnestoLoma) [reply .conda-envs/meta.yaml:58]: done

File: .conda-envs/meta.yaml (hunk 3 above)
Comment (ErnestoLoma) [reply .conda-envs/meta.yaml:65]: done

**Review by Aryamanz29 — APPROVED** [review]
LGTM :rocket:

**Pull request merged**